src/api/migraphx.py

```diff
@@ -193,7 +193,7 @@ def operation(h):
     h.constructor('create',
                   api.params(name='const char*', attributes='const char*'),
                   fname='migraphx::create_op')
-    h.method('name', invoke='migraphx::op_name($@)', returns='const char*')
+    h.method('name', invoke='migraphx::op_name($@)', returns='std::string&')
```

Comment:
This needs to be `std::string` without the `&` because it returns by value.

Reply:
Also, `invoke='migraphx::op_name($@)'` is not needed because it will by default call `op.name()`.

Reply:
fixed